README.md

```diff
@@ -286,27 +286,25 @@ Nyxie.Serialization --> \build\website
 Sitecore.Commerce.Engine --> \build\commerce
 
 - Build Sitecore XC Docker images according to the instructions found here: https://github.com/Sitecore/docker-images
-    - Or if you have pre-built Docker images available in a registry, set the `REGISTRY` in (./env)
-
-- Copy your Sitecore license file (license.xml) to the [](./license) folder
```

Comment:
Hoezo is deze regel eruit gehaald? De license is toch nog steeds nodig?

Reply:
Licensing gebruik is aangepast in community Docker repo. Heb het [hierin](https://github.com/avivasolutionsnl/Nyxie/pull/28/commits/a989d60bd548b565e3f6f7e0589e6b4f7a1c2f0a) explicieter gemaakt.